ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/services/admin/ApiPackageManagementService.scala

```diff
@@ -76,6 +76,10 @@ class ApiPackageManagementService(
           .flatMap {
             case UploadPackagesResult.Ok =>
               Future.successful(UploadDarFileResponse())
+            case r @ UploadPackagesResult.Overloaded =>
```

Comment:
```suggestion
            case UploadPackagesResult.Overloaded =>
```
`r` seems to be unused.